.nycrc

```diff
@@ -1,7 +1,7 @@
 {
     "cache": false,
     "check-coverage": true,
-    "branches": 93,
+    "branches": 92,
```

Comment:
what is this change doing?

Reply:
it's making `npm run cover` pass. https://github.com/kontist/js-sdk/pull/252/commits/6a5e467404a18d4e77bdb8bf4c33491a3ee00d97